assets/app/scripts/controllers/catalog/images.js

```diff
@@ -25,7 +25,6 @@ angular.module('openshiftConsole')
               imageRepoTag: imageRepoTag,
               name: imageRepo.metadata.name + ":" + imageRepoTag
             };
-            $scope.images.push(image);
```

Comment:
@liggitt yes. I had chatted with @jwforres a while back about the possibility of creating a new endpoint in the master that would collect all that info and return it via single request, but we haven't done that and I'm not sure how @smarterclayton would feel about that.